charts/incubator/alfresco-api-gateway/templates/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "alfresco-api-gateway.fullname" . }}
+  labels:
+    app: {{ template "alfresco-api-gateway.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "alfresco-api-gateway.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      imagePullSecrets:
+        - name: {{ .Values.registryPullSecrets }}
```

Comment:
This will cause the chart to fail because you removed the value entirely, either you remove this too or leave the value empty.

Reply:
Opps!  Missed that. Removing it now